raspi/controlserver/controlserver.py

```diff
@@ -21,7 +23,17 @@
 
 wifi_search_folder = os.path.join(dir_path, '../access_point')
 susiconfig = '/home/pi/SUSI.AI/bin/susi-config'
-
+cfg = susi_config.SusiConfig(dir_path + "/../../..")
+
+def get_token():
+    url = 'http://api.susi.ai/aaa/login.json?'
```

Comment:
Please remove the redundant `?` character.

Reply:
updated